### DISCUSSION TITLE
Where to download latest version of DB3K

### DISCUSSION THREAD

**Discussion by Siegemiester** [PygmalionOfCyprus/cmo-db-requests#4428] [Q&A]
So I have the steam version of Command and it shows the latest version of DB3K to be V504, Yet I've seen some people on this board talk about having access to V505.   Is there a non-Steam site that is distributing the latest version of DB3K?

**Comment by PygmalionOfCyprus** [comment]
https://www.matrixgames.com/forums/viewtopic.php?f=10201&t=402322
